pkg/proxy/conntrack/conntrack.go

```diff
@@ -54,6 +56,15 @@ func newConntracker(handler netlinkHandler) Interface {
 	return &conntracker{handler: handler}
 }
 
+// ListEntries list all conntrack entries for connections of the given IP family.
+func (ct *conntracker) ListEntries(ipFamily uint8) ([]*netlink.ConntrackFlow, error) {
+	if entries, err := ct.handler.ConntrackTableList(netlink.ConntrackTable, netlink.InetFamily(ipFamily)); err != nil {
+		return nil, err
+	} else {
+		return entries, nil
+	}
```

Comment:
nit, is this not the same as  `return ct.handler.ConntrackTableList(netlink.ConntrackTable, netlink.InetFamily(ipFamily))`

Reply:
oops.